src/Timing.cc

```diff
@@ -385,4 +385,44 @@ float Timing::dynamicPower(odb::dbInst* inst, sta::Corner* corner)
   return (power.internal() + power.switching());
 }
 
+void Timing::makeEquivCells()
+{
+  sta::Network* network = cmdLinkedNetwork();
+  sta::LibertyLibrarySeq libs;
+  sta::LibertyLibraryIterator* lib_iter = network->libertyLibraryIterator();
```

Comment:
using std::unique_ptr on the iterators makes the deletes much safer. That way you don't have to manually manage the delete.

`std::unique_ptr<sta::LibertyLibraryIterator> lib_iter = std::unique_ptr(network->libertyLibraryIterator()); `

Reply:
fix that!